connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -298,6 +304,8 @@ protected static ConfigDef baseConfigDef() {
                         Importance.LOW, CONFIG_PROVIDERS_DOC)
                 .define(REST_EXTENSION_CLASSES_CONFIG, Type.LIST, "",
                         Importance.LOW, REST_EXTENSION_CLASSES_DOC)
+                .define(ADMIN_LISTENERS_CONFIG, Type.LIST, null,
+                        Importance.LOW, ADMIN_LISTENERS_DOC)
```

Comment:
Do we need a validator here that ensures that the values are one of the following?
1) null
2) an empty list
3) a list of non-empty strings